api/src/main/java/org/openmrs/api/db/hibernate/HibernateProgramWorkflowDAO.java

```diff
@@ -295,16 +299,13 @@ public ConceptStateConversion getConceptStateConversionByUuid(String uuid) {
 		return (ConceptStateConversion) sessionFactory.getCurrentSession().createQuery(
 		    "from ConceptStateConversion csc where csc.uuid = :uuid").setString("uuid", uuid).uniqueResult();
 	}
-	
-	/**
-	 * @see org.openmrs.api.db.ProgramWorkflowDAO#getPatientProgramByUuid(java.lang.String)
-	 */
+
 	@Override
 	public PatientProgram getPatientProgramByUuid(String uuid) {
-		return (PatientProgram) sessionFactory.getCurrentSession().createQuery(
```

Comment:
Would be nice to leave existing code the way it is that is not related to this ticket